src/theme.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+require('babel-polyfill');
```

Comment:
Is `babel-polyfill` needed here? This is only executed server-side so I'm not sure it's needed. Unless I'm missing something?